### ISSUE TITLE
PB blocks localstorage resources after 4 loads, not 3

### ISSUE THREAD

**Issue by Hainish** [EFForg/privacybadgerfirefox#726]
Tested on https://pbtest.org/ using a fresh Firefox profile with PB newly installed, I get the following result for loading of localstorage objects:

![screenshot - 02102016 - 05 46 10 pm](https://cloud.githubusercontent.com/assets/156137/12967701/3a4cbdf4-d01e-11e5-8c78-3814ffeafa8d.png)

This causes the following test to fail initially:
![screenshot - 02102016 - 05 47 22 pm](https://cloud.githubusercontent.com/assets/156137/12967719/6bff8516-d01e-11e5-9923-d75f8eb874a7.png)

After a page reload, the test passes.

**Comment by cooperq** [comment]
If I recall this might have been necessary due to the timing of how privacy badger detects local storage tracking. @gunesacar may be able to offer more insight. Is it possible to modify the tests to account for this?

**Comment by Hainish** [comment]
@cooperq it's possible.  I'll wait for @gunesacar's reply before I start on this.

**Comment by gunesacar** [comment]
Good catch! Apparently we undercount the supercookie prevalence by one.

The reason for this is we don't call `recordPrevalence` for the first occurrence of supercookie trackers, since we record the prevalence in [heuristicBlockingAccounting](https://github.com/EFForg/privacybadgerchrome/blob/3b32d0a61d6a45500b53e428d913b32fcbe0cb65/src/heuristicblocking.js#L592) and we don't yet know whether this is a supercookie tracker or not. 

I added a call to `recordPrevalence` when we first encounter a supercookie tracker.
https://github.com/EFForg/privacybadgerchrome/compare/master...gunesacar:issue726?expand=1 
The test passes with this change.

This issue applies to both Firefox and Chrome versions, if you like the Chrome fix I can send PR for Firefox too.
